drivers/modem/hl7800.c

```diff
@@ -1783,10 +1783,16 @@ static bool on_cmd_atcmdinfo_iccid(struct net_buf **buf, uint16_t len)
 		LOG_INF("EID: %s", delim + 1);
 	}
 
+/* Silence format-truncation warnings when logs are not enabled, as the
+ * compiler seems to think that snprintk tructation is not handled.
```

Comment:
```suggestion
 * compiler seems to think that snprintk truncation is not handled.
```

wondering if this approach will scale well...

Reply:
thanks, fixed, there were only 2 warnings for this in the weekly run and one we were able to handle it by refactoring one of the functions, so there's hope?